### DISCUSSION TITLE
Commissions and borrower account number

### DISCUSSION THREAD

**Discussion by nitins11** [ONDC-Official/ONDC-FIS-Specifications#8] [General]
1. How are commissions being communicated between buyer app and seller app.
2. Is account number fetch of the borrower being done on the lender's form or buyer app's form? If it is buyer app, will buyer app have the liability of a wrong account number shared with the seller app?

**Comment by Antriksh-Parmar** [comment]
1. Commissions will be communicated over API itself. @92shreyansh could you pls add it to the Github?
2. Borrower will enter the account number on the buyer application and BAP will share it with the lender. Lender will then perform 2 checks, penny drop and verification of the last 4 digits of the acc. number with the one that came from AA. The liability's not on the BAP. Besides, we can also supply the last 4 digits of the acc number of the lender to the buyer app so that one check can be done on the BAP as well.

**Reply by 92shreyansh** [reply]
Hi @Antriksh-Parmar will update the github illustartion to include Buyer Finder Fees as part of the specifications
